luma/core/sprite_system.py

```diff
@@ -122,6 +124,9 @@ def __len__(self):
         """
         return self.frames.count
 
+    def close(self):
+        self._img.close()
+
```

Comment:
unfortunately self.image.close() didn't work so added a new private var that can be closed.